official_models/ev3/education_expansion/stair_climber/main.py

```diff
@@ -59,29 +67,29 @@
 # variable.  It repeats until the Center Button is pressed.
 while True:
     # Display the steps variable on the screen.
-    brick.display.clear()
-    brick.display.text(steps, (90, 70))
+    ev3.screen.clear()
+    ev3.screen.draw_text(70, 50, steps)
     wait(200)
 
     # Wait until any Brick Button is pressed.
-    while not any(brick.buttons()):
-        wait(10)
+    while not any(ev3.buttons.pressed()):
+        wait(100)
```

Comment:
Any reason to change to 100? Same for change from `pass` to 100 further above.

Reply:
I should probably go back through and make them all consistent. I think different programs used different values.